apps/docs/content/guides/auth/passwords.mdx

```diff
@@ -122,7 +122,7 @@ You can optionally specify a URL to redirect to after the user clicks the confir
 If you don't specify a redirect URL, the user is automatically redirected to your site URL. This defaults to `localhost:3000`, but you can also configure this.
 
 ```python
-data = await supabase.auth.sign_up({
+data = client.auth.sign_up({
```

Comment:
```suggestion
data = supabase.auth.sign_up({
```

Our convention across the docs for all client libraries is to refer to the client as `supabase`, but I think you're totally right about the await